src/app/zap-templates/templates/app/cluster-typedefs-check.zapt

```diff
@@ -0,0 +1,29 @@
+{{> header}}
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-typedefs.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+{{#zcl_typedefs}}
+{{#if has_more_than_one_cluster}}
+{{> cluster_typedefs_ensure_known_value ns="detail"}}
```

Comment:
Where is cluster_typedefs_ensure_known_value defined?  What is it supposed to do?

Reply:
Apologies, was copy-pasting enum as a template, as you suggested this is unneeded (we don't need to generate unknown value for a typedef)